se agrega botton para agregar cantidad

danyvalenz opened the pull request:
botton que permite ingresar una cantidad

danyvalenz committed:
se agrega botton para agregar cantidad

botton que permite ingresar una cantidad

davidlondono27 commented on Currency App/Currency App/Presentation/UserDetails/View/ContentView.swift:
```diff
@@ -70,8 +77,12 @@ struct ContentView: View {
                     .listStyle(.inset) // -> Este es el estilo de lista, pueden cambiarlo por ejemplo a .automatic para que vean la diferencia
                 Text("\(ConstantsText.currentSearch) \(query.amount) \(origenCurrency) \(ConstantsText.fromAndTo) \(destinoCurrency).")
                     .multilineTextAlignment(.center)
+                TextField("ingrese cantidad", value: $amount, formatter: formatter)
```
Agregar textos a Constantes por favor

davidlondono27 commented on Currency App/Currency App/Presentation/UserDetails/View/ContentView.swift:
```diff
@@ -11,11 +11,18 @@ struct ContentView: View {
     @StateObject var calculatorViewModel : currencyViewModel = currencyViewModel()
     @StateObject var symbolViewModel : symbolsViewModel = symbolsViewModel()
     @State var result = ""
+    @State var amount = 1
```
Declarar variable en 0 por favor

danyvalenz committed:
correcciones de texto a constant_text

se realizan los cambios de inicializar la variable a 0 y de  remplazar todas las strings por constantes

danyvalenz commented:
Se realizaron los cambios solicitados
